MageOS/AsyncEventsAWS/Plugin/MageOS/AsyncEventsAdminUi/UiSourceNotifiers.php

```diff
@@ -0,0 +1,16 @@
+<?php
+namespace MageOS\AsyncEventsAWS\Plugin\MageOS\AsyncEventsAdminUi;
+
+use MageOS\AsyncEventsAdminUi\Ui\Source\Notifiers as Subject;
+
+class UiSourceNotifiers
+{
+    public function afterToOptionArray(Subject $subject, $result)
+    {
+        $result[] = [
+            'value' => 'event_bridge',
```

Comment:
the key is `eventbridge`

ref: https://github.com/mage-os/mageos-async-events-aws/blob/main/etc/di.xml#L7